corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -93,15 +93,23 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
           </td>
           {% if fhir_integration_enabled %}
           <td>
-            <input class="form-control" data-bind="value: $data.fhirResourcePropPath"></input>
+            <input class="form-control" data-bind="value: $data.fhirResourcePropPath, disable: removeFHIRResourcePropertyPath"></input>
```

Comment:
note: This simply disables the input field for path so that user can't edit it after it has been marked to be removed. Also kinda highlights that the field is now going to be removed.

we also set it back to it's original value using 
`self.fhirResourcePropPath(self.originalResourcePropPath);`
so that we delete the correct corresponding entry.
It's possible user would edit the path and then mark it for removal, but we need the original value. 
We could have simply passed the original value in ajax request on save and still kept the changed value here, in case user changes it before marking for removal, but thought to reset it so that it's not implicit and user knows what they are "actually" removing irrespective of their change done to the path.